### DISCUSSION TITLE
Doesn't seem to work with Star rails

### DISCUSSION THREAD

**Discussion by myxgenius** [TKMAX777/RDPRelativeInput#7] [General]
After following the usage instruction, the mouse on my rdp session does changed and it feel something changed. However when i start Star Rails game, which should be similar to genshin impact, i can only turn about 180 degree from left to right. It is as if the mouse has reached the edge of screen which limits the turning. This is the same behavior when i'm not using this program. When I select the RDP Input Wrapper window and hit the F8 key, the mouse just no longer works.

**Comment by TKMAX777** [comment]
Thank you for asking me. 

I want to confirm whether you already execute RelativeInputServer.exe with administrator privileges or not. If the game's behaviours are the same as Genshin impact, that problem will be solved with this.
